language-support/ts/codegen/src/TsCodeGenMain.hs

```diff
@@ -661,22 +661,22 @@ instance ToJSON PackageJson where
 -- Read the provided 'package.json'; transform it to include the
 -- provided workspaces; write it back to disk.
 setupWorkspace :: FilePath -> [(T.Text, [Dependency])] -> FilePath -> IO ()
-setupWorkspace optOutputDir dependencies file = do
+setupWorkspace optOuputDir dependencies file = do
   let (g, nodeFromVertex) = graphFromEdges'
         (map (\(a, ds) -> (a, a, map unDependency ds)) dependencies)
       ps = map (fst3 . nodeFromVertex) $ reverse (topSort g)
         -- Topologically order our packages.
-      scope = scopeOfScopeDir optOutputDir
-        -- 'scope' we expect to be something like "daml2ts".
-  let ourWorkspaces = map ((unScope scope <> "/") <>) ps
+      scopeDir = T.pack $ takeFileName optOuputDir
```

Comment:
Does anything break if I set `optOutputDir` to something different than the scope? Maybe a good idea to have a testcase for that?

Reply:
There is no relationship between scope and the output directory. For example, the DAVL unit test defaults the scope to `@daml.js` but the output directory is `daml2ts`.

Reply:
Then maybe this should not be called `scopeDir` given that it’s completely independent of the scope?

Reply:
well, it's the scope dir in that it's the directory that contains all the packages that are associated with the scope

Reply:
i could rename it to say, `leafDir` or `outDir` or something if you think it confusing?

Reply:
Maybe `outBaseName` or something like that?

Reply:
ok. will do.